packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -20,17 +20,23 @@ export interface ISummaryStats {
 }
 
 export interface ISummaryTreeWithStats {
-    stats: ISummaryStats;
+    stats: ITopLevelSummaryStats;
     summary: ISummaryTree;
 }
 
+export interface ITopLevelSummaryStats extends ISummaryStats {
+    dataStoreCount: number;
+    summarizedUnreferencedDataStoreCount: number;
+    summarizedDataStoreCount: number;
```

Comment:
nit: Simpler names could be - `dataStoreTreeCount` and `unreferencedDataStoreTreeCount`